### PR TITLE
Adding upstream pytorch installation instruction for Intel GPUs to README.md 

### PR DESCRIPTION
**Pull request by qiacheng**
updated and simplfied ComfyUI readme for Intel GPUs.

Intel GPU Support Now Available in PyTorch 2.5 https://pytorch.org/blog/intel-gpu-support-pytorch-2-5/

more info can be found: https://pytorch.org/docs/main/notes/get_start_xpu.html

1. using upstream pytorch nightly builds for Intel Arc GPUs, (Desktop discreet A-Series/B-Series GPUs, Core Ultra-H, Core Ultra Series-2)

```
pip install --pre torch torchvision torchaudio --index-url https://download.pytorch.org/whl/nightly/xpu
```

```
python main.py
```

2. Users can also use Intel® Extension for PyTorch (IPEX) to setup env for ComfyUI. This is device specific for improved performance and requires a conda env.

more info can be found: https://pytorch-extension.intel.com/installation?platform=gpu

oneAPI base toolkit setup is no longer needed starting in IPEX 2.3.110+ , using the pip command to install ipex will also pull the dependent oneAPI packages from pypi.

For Intel® Arc™ A-Series Graphics utilizing IPEX, use the commands below:
```
conda install libuv
pip install torch==2.3.1.post0+cxx11.abi torchvision==0.18.1.post0+cxx11.abi torchaudio==2.3.1.post0+cxx11.abi intel-extension-for-pytorch==2.3.110.post0+xpu --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/us/ --extra-index-url https://pytorch-extension.intel.com/release-whl/stable/xpu/cn/
```

Cheers! :)

### CODE REVIEW

**Review by huchenlei — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -149,6 +149,30 @@ This is the command to install the nightly with ROCm 6.2 which might have some p
...
+
+2. Launch ComfyUI by running `python main.py`
+
+
+(Option 2) Alternatively, Intel GPUs supported by Intel Extension for PyTorch (IPEX) can leverage IPEX for improved performance.
+
+1. For Intel® Arc™ A-Series Graphics utilizing IPEX, create a conda environment and use the commands below:
+
+```
+conda install libuv
```
Comment (comfyanonymous) [inline README.md:168]: Is this step necessary?

Comment (qiacheng) [reply README.md:168]: It is necessary if users want to use Intel Extension for Pytorch for better performance on Intel GPUs. 

Currently there is a perf gap in Pytorch 2.5 / Pytorch 2.6 nightly vs. Pytorch 2.3 (also perf regression on Nvidia/AMD too, as reported in https://github.com/pytorch/pytorch/issues/138386)

so it's good to have both options available for Intel GPU users, and show an example of the env setup.

Eventually we would want to make an update to ComfyUI readme when those issues are fixed 😄 

Thanks

Comment (qiacheng) [reply README.md:168]: conda install libuv is necessary as libuv is one of the required dependencies of ipex.

https://pytorch-extension.intel.com/installation?platform=gpu&version=v2.3.110%2Bxpu&os=windows&package=pip

Comment (webfiltered) [reply README.md:168]: > A conda environment is required.

Looks like they only support conda w/libuv on Windows - linux/WSL2 has more flexibility.

**Pull request merged**